### PR TITLE
Frontend select role dev or cust and customer base info  pages

### PR DESCRIPTION
**Pull request by middaysan**

### CODE REVIEW

File: app/controllers/home_controller.rb [hunk 1]
```diff
@@ -1,6 +1,17 @@
 # frozen_string_literal: true
 
 class HomeController < ApplicationController
-  # def index
-  # end
+  before_action :handle_unfilled_rol, only: [:index]
+
+  def index; end
+
+  def handle_unfilled_rol
```
Comment (andreydro) [inline app/controllers/home_controller.rb]: `handle_unfilled_role`?

File: app/controllers/profile_controller.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+  def customer_setup_info; end
+
+  def developer_setup_info; end
+
+  # This module smells of :reek:NilCheck
+  def update_role
+    role = params[:role]&.to_sym
+    raise "Unknown role: #{role}" unless User::USER_ROLES.key?(role)
+    @user.update(role: role)
```
Comment (andreydro) [inline app/controllers/profile_controller.rb]: What if update fail?

File: app/controllers/home_controller.rb (hunk 1 above)
Comment (middaysan) [reply app/controllers/home_controller.rb]: typo error, fix

File: app/controllers/profile_controller.rb (hunk 2 above)
Comment (middaysan) [reply app/controllers/profile_controller.rb]: add if block and else redirect_to back to role choice

**Pull request merged**